code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/EdgePersonalizationResponseHandler.java

```diff
@@ -583,29 +586,43 @@ private void updateRulesEngines(
                     launchRulesEngine.replaceRules(collectedInAppRules);
                     break;
                 case FEED:
```

Comment:
nit: Do we still need `case FEED:` ?

Reply:
Nvm, noticed that we only deprecated Feed, didn't remove it completely

Reply:
no, i guess not - i was thinking it was needed for backwards compatibility but nobody is creating `feed-item` schema objects in production today